BrainPortal/app/views/tasks/show.html.erb

```diff
@@ -47,6 +47,17 @@
 <% if @task.bourreau.online? %>
 
   <div class="menu_bar">
+    <!-- TASKS IN BATCH -->
```

Comment:
Can you put the link into the Info section.

Reply:
I moved the link in the summary section instead because that's where it was in the other files that contained the broken links.